src/main/java/address/controller/MainController.java

```diff
@@ -146,7 +146,13 @@ public StatusBarHeaderController getStatusBarHeaderController() {
 
     private void showHeaderStatusBar() {
         statusBarHeaderController = new StatusBarHeaderController();
-        rootLayout.getChildren().add(2, statusBarHeaderController.getFooterStatusBarView());
+        AnchorPane sbPlaceHolder = (AnchorPane) rootLayout.lookup("#footerStatusbarPlaceHolder");
+        if (sbPlaceHolder == null) {
+            logger.warn("Error finding layout node");
+            showAlertDialogAndWait(AlertType.ERROR, "Error loading layout resources", "", "");
+        }
```

Comment:
Yup right, changed it. 

I guess we need to change it to assertion for error in loading FXML also.

Will create another PR for changing the FXML one.